src/anim/evaluator/anim-evaluator.js

```diff
@@ -149,6 +276,19 @@ class AnimEvaluator {
                     }
 
                     targets[resolved.targetPath] = target;
+                    if (this._binder.animComponent) {
+                        if (!this._binder.animComponent.targets[resolved.targetPath]) {
+                            let type;
+                            if (resolved.targetPath.indexOf('localRotation') === -1) {
```

Comment:
does the targetPath store the entity names path as well? If yes, having an entity called "localRotation" would not work well with this.